src/html/HTMLWriter.jl

```diff
@@ -1854,13 +1854,13 @@ function domify(dctx::DCtx, mdast_node::Node, docsnode::Documenter.DocsNode)
     )
     push!(ctx.search_index, rec)
 
-    return article[".docstring"](
-        header(
-            a[".docstring-article-toggle-button.fa-solid.fa-chevron-down", :href => "javascript:;", :title => "Collapse docstring"],
```

Comment:
Visually, it would be nice if we could retain the `fs-chevron-*` symbols for the triangle. Not sure if that's easy to do with CSS or not.